vector/src/main/java/im/vector/app/core/utils/UrlUtils.kt

```diff
@@ -38,6 +38,13 @@ internal fun String.ensureProtocol(): String {
     }
 }
 
+/**
+ * Ensure string do not starts with "http" or "https" protocol.
+ */
+internal fun String.ensureNoProtocol(): String {
```

Comment:
`internal` is a bit useless here

Reply:
Used same qualifier than other methods in the file, but I guess it's some copy/paste from SDK, will change

Reply:
Finally removed the method, I do not need it anymore. I still removed the internal keywords from the file